src/esperanto/ProjectBundle/Resources/public/js/base.js

```diff
@@ -0,0 +1,62 @@
+
```

Comment:
Which one is used, the NewsletterForm,js or the base.js?
Make sure there is only one js to include and better we call it NewsletterForm,js cause this name inform more about the context.